src/App.jsx

```diff
@@ -1,12 +1,32 @@
+import "./App.css";
+import { useState, useEffect, useContext } from "react";
+import Header from "./components/Header.jsx";
+import CategoryFilter from "./components/CategoryFilter.jsx";
+import RestaurantList from "./components/RestaurantList.jsx";
+import AddRestaurantModal from "./components/AddRestaurantModal.jsx";
+import RestaurantDetailModal from "./components/RestaurantDetailModal.jsx";
+import { getRestaurantList } from "./api/restaurantAPI";
 
-import './App.css'
+import { CategoryProvider } from "./contexts/CategoryContext";
+import { RestaurantListProvider } from "./contexts/RestaurantListContext";
+import { ModalContext, ModalProvider } from "./contexts/ModalContext";
 
 function App() {
+  const { modal, isAddModalOpen } = useContext(ModalContext);
+
   return (
     <>
-        Code It..
+      <Header />
+      <main>
+        <CategoryFilter />
+        <RestaurantList />
+      </main>
+      <aside>
+        {modal.isOpen && <RestaurantDetailModal />}
+        {isAddModalOpen && <AddRestaurantModal />}
```

Comment:
모달이 두 개여서 이 두개를 `addModal`과 `detailModal`처럼 구분할 수 있는 이름을 사용해도 좋을 것 같아요!
```suggestion
        {detailModal.isOpen && <RestaurantDetailModal />}
        {addModal.isOpen && <AddRestaurantModal />}
```